src/FSharp.Core/list.fsi

```diff
@@ -1221,6 +1371,9 @@ module List =
     [<CompiledName("IterateIndexed2")>]
     val iteri2: action:(int -> 'T1 -> 'T2 -> unit) -> list1:'T1 list -> list2:'T2 list -> unit
 
+    /// 
+    /// Time complexity: O(n) - where n is the list length
+    /// Space complexity: O(1)
```

Comment:
The added complexity information should be within the `<summary>` tag, right before the closing tag.